src/node.ts

```diff
@@ -0,0 +1,59 @@
+import { PaymentSdkConfigType } from "./types";
+
+import { XcooBeePaymentSDK } from "./XcooBeePaymentSDK";
+
+import { logoSrc } from "./config";
+
+/**
+ * Implements node version of Base64Interface to manipulate base64 strings
+ *
+ * @implements Base64Interface
+ */
+export const base64 = {
+  atob: (str: string): string => Buffer.from(str, "base64").toString(),
+  btoa: (str: string): string => Buffer.from(str).toString("base64"),
+};
+
+/**
+ * Implements node version of QrGeneratorInterface to generate QRs
+ *
+ * @implements QrGeneratorInterface
+ */
+export const qrGenerator = {
```

Comment:
Will the following work to help enforce it maintains the correct interface?
```suggestion
export const qrGenerator: QrGeneratorInterface = {
```